package.json

```diff
@@ -13,23 +13,22 @@
 	},
 	"dependencies": {
 		"eslint-config-airbnb-base": "^14.1.0",
-		"eslint-config-prettier": "^6.11.0",
+		"eslint-config-prettier": "^7.2.0",
 		"eslint-plugin-compat": "^3.7.0",
 		"eslint-plugin-eslint-comments": "^3.2.0",
 		"eslint-plugin-filenames": "^1.3.2",
 		"eslint-plugin-import": "^2.21.1",
 		"eslint-plugin-jest": "^24.1.0",
 		"eslint-plugin-jsx-a11y": "^6.2.1",
-		"eslint-plugin-mocha": "^8.0.0",
 		"eslint-plugin-no-secrets": "^0.6.8",
 		"eslint-plugin-optimize-regex": "^1.2.0",
 		"eslint-plugin-prettier": "^3.1.3",
 		"eslint-plugin-promise": "^4.1.1",
 		"eslint-plugin-react": "^7.20.0",
-		"eslint-plugin-simple-import-sort": "^5.0.3",
+		"eslint-plugin-simple-import-sort": "^7.0.0",
 		"eslint-plugin-sonarjs": "^0.5.0",
 		"eslint-plugin-sort-keys-fix": "^1.1.1",
-		"eslint-plugin-unicorn": "^22.0.0"
+		"eslint-plugin-unicorn": "^27.0.0"
```

Comment:
Sidenote on this dep. For some reason, this guy sometimes pushes breaking changes to this repo as non-major versions. Maybe he's stopped, and it's easy to revert and find the new rule that's mangling the code, but it's bitten me a couple times. So just to note.

It's weird, because he's a super-popular Node guy, and all his other stuff is top notch. I guess he thinks linting is not major or something.

Reply:
Maybe "breaking" is too strong. But he'll introduce new rules in minor versions that when used with `--fix` will rewrite giant amounts of your codebase.